components/tests/python/library/__init__.py

```diff
@@ -116,6 +116,18 @@ def teardown_class(cls):
         cls.root = None
         cls.__clients.__del__()
 
+    def keepRootAlive(self):
+        """
+        Keeps root connection alive.
+        """
+        try:
+            if self.root.sf is None:
+                assert self.root.connect()
```

Comment:
Is `root` here a `BlitzGateway` or a `BaseClient`? In the latter case, I don't think there's a "connect" method. NB: an alternative option would be [enableKeepAlive](https://github.com/openmicroscopy/openmicroscopy/blob/v5.2.2/components/tools/OmeroPy/src/omero/clients.py#L612) on the `BaseClient` but that will require cleaning them up, otherwise tests may hang.